week3/homework/project/src/Components/CitySearch.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+
+import magnifierIcon from './imgs/magnifier.png'
+
+function CitySearch(props) {
+    const [input, setInput] = useState({ name: "" });

```

Comment:
Why did you nested the value here under the prop name? and not like
```suggestion
    const [input, setInput] = useState("");

```